steps/testproject-upload/0.1.0/step.yml

```diff
@@ -0,0 +1,74 @@
+title: Upload a file to TestProject
+summary: Upload an IPA/APK file to TestProject
+description: Upload an IPA/APK file to TestProject
+website: https://github.com/sca-organization/bitrise-testproject-upload
+source_code_url: https://github.com/sca-organization/bitrise-testproject-upload
+support_url: https://github.com/sca-organization/bitrise-testproject-upload/issues
+published_at: 2022-03-24T12:45:55.639574+11:00
+source:
+  git: https://github.com/sca-organization/bitrise-testproject-upload.git
+  commit: 4b32da488eebd3a2ef77c3f5c9ac2a0f5e186c5c
+project_type_tags:
+- ios
+- android
+- macos
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
+is_requires_admin_user: true
```

Comment:
Please remove _is_requires_admin_user_, it is deprecated.

Reply:
thanks. removed